service/grpc/codelingo/codelingo.pb.go

```diff
@@ -1336,6 +1000,32 @@ func (x *codeLingoReviewServer) Recv() (*ReviewRequest, error) {
 	return m, nil
 }
 
+func _CodeLingo_Codemod_Handler(srv interface{}, stream grpc.ServerStream) error {
+	return srv.(CodeLingoServer).Codemod(&codeLingoCodemodServer{stream})
+}
+
+type CodeLingo_CodemodServer interface {
```

Comment:
don't use underscores in Go names; type CodeLingo_CodemodServer should be CodeLingoCodemodServer